hedyorg/hedy
# [BUGFIX] Fixes keyword languages not changing when changing site's language

First, provide a one line summary of your changes in the **Title of the PR**

**Now fill out the remainder of this template by replacing all _italic_ content**

**Description**
As stated in #3433 when one switches the language of the site in the dropdown menu, if one changed the keyword language to any language other than English, when switching to other language the keyword languages stayed the same. This PR fixes that, by changing the `keyword_language` parameter to English when it's present.

**Fixes #3433**

**How to test**
1. Go to the Hedy editor
2. Switch the language and the keyword language.
3. Switch to another language.

You should see the keywords in English

**Checklist**
Done? Check if you have it all in place using this list:*
  

If you're unsure about any of these, don't hesitate to ask. We're here to help!
